### PR TITLE
Add Build Script to Include Required Stripe Dependencies 

### PR DESCRIPTION
**Pull request by spencerfinnell**
Proposed Changes:
1. Add `npm run build` script to generate a `.zip` that includes Stripe gateway dependencies.

### CODE REVIEW

**Comment by cklosowski** [comment]
The only issue I see with this is that we're not replacing the text domain of `"edds"` with `"easy-digital-downloads"`. We'll need to be able to generate a `.pot` file that includes all the translatable strings.

Going to see if I can get something into the bash script to do that.

**Comment by cklosowski** [comment]
Nevermind, looks like our grunt task corrects these on the fly. Super!

**Review by ashleyfae — APPROVED** [review]

**Review by robincornett — APPROVED** [review]
Just noting that this works for the Windows user. :)

**Comment by ashleyfae** [comment]
I appreciate you checking that @robincornett !

File: bin/update-stripe.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Remove anything that already exists.
+rm -rf includes/gateways/stripe
+
+# Initial clone
+git clone -b release/2.8.1 git@github.com:easydigitaldownloads/edd-stripe includes/gateways/stripe
```
Comment (ashleyfae) [inline bin/update-stripe.sh:5]: Branch can be updated now.

**Review by ashleyfae — APPROVED** [review]

**Review by cklosowski — APPROVED** [review]

**Pull request merged**